### DISCUSSION TITLE
Change delegation with shortened debonding period

### DISCUSSION THREAD

**Discussion by subscriber123** [oasisprotocol/community-forum#15] [Ideas]
It should be possible to change validators without having to wait the entire 14 day debonding period.

This will increase competition among validators and encourage them to support new paratimes.

I don't see any disincentive to making this instant, but even a 1 day debonding period would be an improvement.

**Comment by subscriber123** [comment]
This is related to https://github.com/oasisprotocol/community/discussions/3 to shorten the debonding period in general, but this suggestion is entirely independent of that.

**Comment by al3-al3** [comment]
Yes please consider reducing the 14 days, it seems excessive especially as you are out of earnings for the period too. Thank you

**Comment by jzlim2** [comment]
I totally agree.

**Comment by arfonzocoward** [comment]
Definitely agree. 14 days is overly onerous for re-delegations.

**Comment by jevinoh** [comment]
+1

7 days or less would be awesome

**Comment by kostko** [comment]
The Foundation is focused on the security of the network first and foremost. Shortening the debonding period or adding support for redelegation seems like a small thing on first sight, but it would add a layer of complexity to the staking part to implement securely.

That being said, if there is enough interest in the community, the Foundation is committed to working with the community to design a spec (ADR) around redelegation and facilitating its implementation that could be proposed in any of the subsequent network upgrades.

**Reply by Irfan22in** [reply]
I see this feature implemented in many other networks simply because it is required for several reasons. This could be due to change in commissions,uptime of validators ,change in the validator ranking (escrow weight). Is there anyway to get the community vote/view on this feature ?

**Comment by vinay2393** [comment]
I think totally agree with the OP;

If you are simply going to switch validators and not remove your ROSE from staking then switching validators should not have to undergo the general 14 days unbonding..

I've had one of my validators double their commission overnight and delegators are the ones paying for it..
Similar issues can be encountered if a certain validator doesn't have a high uptime..the delegator is the one at a loss

I'm not against the 14 days unbonding (other projects like DOT or ATOM have longer unbonding periods) but if you (as a delegator) are continuing to support the network then I think unbonding is irrelevant. The added benefit is that it forces validators to become competitive (both with commission rates and better uptime/service)

**Comment by JODIN1727** [comment]
Hi Team.. my apologies and I am now back to our community discussion further..
